.pipelines/templates/obp-file-signing.yml

```diff
@@ -138,10 +139,17 @@ steps:
 - pwsh: |
     Import-Module '$(PowerShellRoot)/build.psm1' -Force
     Import-Module '$(PowerShellRoot)/tools/packaging' -Force
-    $pathForUpload = New-Item -ItemType Directory -Path '$(ob_outputDirectory)/Signed-$(Runtime)' -Force
-    Write-Verbose -Verbose -Message "pathForUpload: $pathForUpload"
-    Copy-Item -Path '${{ parameters.binPath }}\*' -Destination $pathForUpload -Recurse -Force -Verbose
-    Write-Verbose -Verbose -Message "Files copied to $pathForUpload"
+    $isGlobalTool = '${{ parameters.globalTool }}' -eq 'true'
```

Comment:
Not blocking.  We should move any PS Script of significant size into a script or module.  This does two things, makes sure they are scanned by static analysis tools and reduces total yaml size.